doc/man3/SSL_CONF_cmd.pod

```diff
@@ -162,6 +162,19 @@ This is a synonym for the B<-groups> command.
 This sets the temporary curve used for ephemeral ECDH modes. Only used
 by servers.
 
+=item B<-cert_comp>
+
+Enables support for SSL/TLS compression, same as clearing
```

Comment:
TLS 1.3 (same applies to section below).

Reply:
Fixed